Resources/config/odm.xml

```diff
@@ -4,54 +4,17 @@
     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
     xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
 
-    <parameters>
-
-        <parameter key="doctrine_phpcr.odm.configuration.class">Doctrine\ODM\PHPCR\Configuration</parameter>
-        <parameter key="doctrine_phpcr.odm.document_manager.class">Doctrine\ODM\PHPCR\DocumentManager</parameter>
-
-        <!-- Translation strategies -->
-        <parameter key="doctrine_phpcr.odm.translation.strategy.attribute.class">Doctrine\ODM\PHPCR\Translation\TranslationStrategy\AttributeTranslationStrategy</parameter>
-        <parameter key="doctrine_phpcr.odm.translation.strategy.child.class">Doctrine\ODM\PHPCR\Translation\TranslationStrategy\ChildTranslationStrategy</parameter>
-
-        <!-- cache -->
-        <parameter key="doctrine_phpcr.odm.cache.array.class">Doctrine\Common\Cache\ArrayCache</parameter>
-        <parameter key="doctrine_phpcr.odm.cache.apc.class">Doctrine\Common\Cache\ApcCache</parameter>
-        <parameter key="doctrine_phpcr.odm.cache.memcache.class">Doctrine\Common\Cache\MemcacheCache</parameter>
-        <parameter key="doctrine_phpcr.odm.cache.memcache_host">localhost</parameter>
-        <parameter key="doctrine_phpcr.odm.cache.memcache_port">11211</parameter>
-        <parameter key="doctrine_phpcr.odm.cache.memcache_instance.class">Memcache</parameter>
-        <parameter key="doctrine_phpcr.odm.cache.memcached.class">Doctrine\Common\Cache\MemcachedCache</parameter>
-        <parameter key="doctrine_phpcr.odm.cache.memcached_host">localhost</parameter>
-        <parameter key="doctrine_phpcr.odm.cache.memcached_port">11211</parameter>
-        <parameter key="doctrine_phpcr.odm.cache.memcached_instance.class">Memcached</parameter>
-        <parameter key="doctrine_phpcr.odm.cache.xcache.class">Doctrine\Common\Cache\XcacheCache</parameter>
-
-        <!-- form field factory guesser -->
-        <parameter key="form.type_guesser.doctrine_phpcr.class">Doctrine\Bundle\PHPCRBundle\Form\PhpcrOdmTypeGuesser</parameter>
-
-        <parameter key="doctrine_phpcr.odm.form.path.type.class">Doctrine\Bundle\PHPCRBundle\Form\Type\PathType</parameter>
-
-        <parameter key="doctrine_phpcr.odm.metadata.driver_chain.class">Doctrine\Common\Persistence\Mapping\Driver\MappingDriverChain</parameter>
-        <parameter key="doctrine_phpcr.odm.metadata.annotation.class">Doctrine\ODM\PHPCR\Mapping\Driver\AnnotationDriver</parameter>
```

Comment:
I think we can not remove those as Symfony-Doctrine-Bridge still uses those `.class` parameters. to create the mapping driver stuff

Reply:
i do not see DoctrinePhpcrMappingsPass or the abstract class use any .class parameters. then again, i did not see where the DoctrinePhpcrMappingsPass is even registered at all. did you see where that happens?

Reply:
It was quite hard to find them. They live in AbstractDoctrineExtension of symfony/doctrine-bridge. I greped for anotation or .class the complete parameter id is concatenated in php except .class

Reply:
https://github.com/symfony/doctrine-bridge/blob/master/DependencyInjection/AbstractDoctrineExtension.php#L200 here is an example

Reply:
```

 +
 +        <parameter key="doctrine_phpcr.odm.metadata.annotation.class">Doctrine\ODM\PHPCR\Mapping\Driver\AnnotationDriver</parameter>
 +        <parameter key="doctrine_phpcr.odm.metadata.driver_chain.class">Doctrine\Common\Persistence\Mapping\Driver\MappingDriverChain</parameter>
 +        <parameter key="doctrine_phpcr.odm.cache.array.class">Doctrine\Common\Cache\ArrayCache</parameter>
```
let the tests run again.

Reply:
ah, got it. added the other cache related parameters back as well, as they look used in the extension. this is one of the uglier pieces of symfony code ...

Reply:
I will add a [note](https://github.com/symfony/symfony/issues/24835).